data/campaigns/Son_Of_The_Black_Eye/scenarios/05_To_the_Harbor_of_Tirigaz.cfg

```diff
@@ -237,7 +237,7 @@
 
         [message]
             speaker="Kapou'e"
-            message= _ "Grüü is right. It is not wise to let these undead remain here cutting off our supply lines and path of retreat. In these days of turmoil, we don’t know what lies ahead, so it is best not to leave any threats in our rear."
+            message= _ "Grüü is right. It is not wise to let these undead remain here — cutting off our supply lines and path of retreat. In these days of turmoil, we don’t know what lies ahead, so it is best not to leave any threats in our rear."
```

Comment:
I'm not sure it makes sense to break up this sentence with an em-dash here. Maybe a comma is sufficient?

Reply:
Comma sounds right. Dash adds too much dramatic tone.